docs/compatibility.md

```diff
@@ -93,6 +93,10 @@ These are repos that can be started as a some kind of process, such as a web app
       * Who knows, really - several tests are failing, lots pass ;-)
       * Rake task to [create a test taxon](https://github.com/alphagov/whitehall/blob/master/lib/tasks/taxonomy.rake#L11) for publishing is not idempotent
       * Placeholder images don't work as missing proxy for [/government/assets](https://github.com/alphagov/whitehall/blob/master/app/presenters/publishing_api/news_article_presenter.rb#L133)
+   - ⚠ govuk-attribute-service-prototype
+      * **TODO: Missing support for a webserver stack**
```

Comment:
What does this TODO mean?

Reply:
Ah that's a great point was mostly[ copy pasting what appeared to be convention](https://github.com/alphagov/govuk-docker/pull/415/files#diff-90ba508fd9c179b95ae20ac325190e9fR76-R77) but that's probably a mistake.

Humm, :thinking: let me think. The webstack as far as i can tell is used in correspondance with actual live services, so would allow you to hit the actual content store rather than any local dev versions, right?

Our dev keys should never match the live prod ones for either of these services... so there's not likely to ever be any functionality we could use with the webstack.

I'll check in with Ben if there's any expectation that the webserver stack does anything more, or should be there even if it doesn't work?

Reply:
for now though will remove the line and rebase, I'll amend those docs in a separate PR

Reply:
Done, just replaced with a note saying we've _not_ got support.
if we need it, i'll add it in later.

Reply:
This TODO is used for cases where there isn't a web server set-up for apps. For example, if we look at content-data-admin and imminence they only have a lite service. This limits you to be able to run commands with the app (like open a console) but lacks a stack with nginx to interface with the web application - hence the phrasing webserver stack. For both of these apps there is an app service with the webserver stack so you don't need this todo.

We should think about changing this wording though as I recall it being a source of confusion in the last few days.